packages/editor/src/components/post-actions/actions.js

```diff
@@ -1035,14 +985,13 @@ export function usePostActions( { postType, onActionPerformed, context } ) {
 				userCanCreatePostType &&
 				duplicateTemplatePartAction,
 			isPattern && userCanCreatePostType && duplicatePatternAction,
-			supportsTitle && renamePostActionForPostType,
+			supportsTitle && renamePostAction,
```

Comment:
It looks like some of these need to be added to the dependency array.

Reply:
they are now top level variables so stable references, the lint rule understands that it seems so they don't need to be added to the dependencies.